tests/testthat/test-setInput.R

```diff
@@ -0,0 +1,37 @@
+library(testthat)
```

Comment:
Kudos for adding the firsts R tests! :grin: Let's remove `test-all.R`. We can also get rid of `mockery` from `DESCRIPTION`.